datumaro/plugins/ade20k2020_format.py

```diff
@@ -49,12 +54,15 @@ def _load_items(self, subset):
             LabelCategories())
         path = osp.join(self._path, subset)
 
-        images = [i for i in find_images(path, '.jpg', recursive=True)]
+        images = [i for i in find_images(path, recursive=True)]
 
         for image_path in sorted(images):
             item_id = osp.splitext(osp.relpath(image_path, path))[0]
-            item_annotations = []
 
+            if Ade20k2020Path.MASK_IMAGE_PATTERN.search(item_id):
```

Comment:
Probably, only the basename should be checked. Also, consider using `re.fullmatch` to match the whole name.

Reply:
Yes, I will change the item_id to the basename, but I don't think that we should use `re.,fullmatch` here, because we don't know full name for instance mask (that should be skipped). We only recognize it after parsing the annotation file.